README.md

```diff
@@ -28,10 +28,19 @@ Before using the boilerplate, we recommend you to go through the documentation o
 3. [Web Performance](https://www.aem.live/developer/keeping-it-100)
 4. [Markup, Sections, Blocks, and Auto Blocking](https://www.aem.live/developer/markup-sections-blocks)
 
-## Config Service Setup
+## Initial Setup
+
+The boilerplate assumes you already have an AEM org and will onboard a new site via config service. If you do not have an org, you will need to contact Adobe to have one created, or you can do the following:
+
+1. Upload the [starter content](https://github.com/hlxsites/aem-boilerplate-commerce/releases/tag/starter-content) somewhere (https://da.live, sharepoint, google drive, etc).
+1. Update the mountpoint in the `default-fstab.yaml` to point to your content.
+1. Rename the file to `fstab.yaml` and commit/push the change.
+1. Install the [AEM Code Sync Bot](https://github.com/apps/aem-code-sync)
+
```

Comment:
but this doesn't create the EDS org, just the site config. which is ok, I guess for now. 
but it doesn't allow the user then later to modify the site config. for that they need to add permissions, via the `/.helix/config.xlsx` config